File: full-vineflower/me/ag2s/umdlib/domain/UmdEnd.java
package me.ag2s.umdlib.domain;

import java.io.IOException;
import me.ag2s.umdlib.tool.WrapOutputStream;

public class UmdEnd {
   public void buildEnd(WrapOutputStream wos) throws IOException {
      wos.writeBytes(35, 12, 0, 1, 9);
      wos.writeInt(wos.getWritten() + 4);
   }
}

File: src/main/kotlin/me/ag2s/umdlib/domain/UmdEnd.kt
package me.ag2s.umdlib.domain

import me.ag2s.umdlib.tool.WrapOutputStream

class UmdEnd {
    fun buildEnd(wos: WrapOutputStream) {
        wos.writeBytes(35, 12, 0, 1, 9)
        wos.writeInt(wos.written + 4)
    }
}
